infra/README.md

```diff
@@ -1,3 +1,62 @@
+# Multichain Infrastructure Overview
+
+## Environments:
+- Testnet (Production)
+- Dev (Development)
+
+## Deployment:
+
+### Development
+#### This environment has been automated for deployment, simply make a pull request with your changes to the `develop` branch, get it reviewed, and merge the PR.
+  - Deployment steps:
+    1. A merged PR triggers the following Github Actions Workflows:
+      - [multichain-dev.yml](../.github/workflows/multichain-dev.yml)
+      - [deploy-multichain-dev-contract.yml](../.github/workflows/deploy-multichain-dev-contract.yml)
+    2. These workflows deploy a new imaged based off of the github SHA tag to the 8 GCP vms and restart the VM
+    3. Then, the smart contract for the dev environment is reinitialized
+
+### "Break Glass" Deployment of Development environment
+#### This should only be used if the environment is completely broken
+  - Deployment steps:
+    1. Make sure you have [terraform installed](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli) on your local machine
+    2. Navigate to the `infra` directory, and then the `multichain-dev` directory
+    3. Verify the variables in both `variables.tf` and `terraform-dev.tfvars` are up to date
+    4. Verify the environment variables (`main.tf lines 17-61`, `variables.tf lines 87-150`) for the container are as desired
```

Comment:
I would not mention lines here, we will forget to update it. I think it's better to mention specific fields you need to update.

Reply:
Feel free to add those, I am not sure what fields get updated frequently.